Library/Formula/dspdfviewer.rb

```diff
@@ -0,0 +1,21 @@
+class Dspdfviewer < Formula
+  desc "Dual-Screen PDF Viewer for latex-beamer"
+  homepage "http://dspdfviewer.danny-edel.de"
+  url "https://github.com/dannyedel/dspdfviewer/archive/v1.13.1.tar.gz"
+  sha256 "333588de0316cfdb5821b8484ee55690dfa3c7431b67c126bfdbe9c9cc3e1ed4"
+
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
+  depends_on "poppler" => "with-qt"
+
+  def install
+    mkdir "build" do
+      system "cmake", ".."
```

Comment:
Please pass `*std_cmake_args` here as well. You're currently trying to install directly to `/usr/local`.